### PR TITLE
Feature: pass filters to vector layer

### PR DESCRIPTION
**Pull request by sorodrigo**

### CODE REVIEW

**Comment by sorodrigo** [comment]
I added the necessary changes to filter years using mapbox styles instead of carto queries.

See example: https://docs.mapbox.com/mapbox-gl-js/example/timeline-animation/

**Comment by kfilip94** [comment]
This PR enables filtering species distribution layers by country, scenario, year or selected species. It adds also current distribution layer to the `Map distribution` view:
[PREVIEW](https://copernicus-forest-git-feature-pass-filters-to-vector-layer.vizzuality1.now.sh)
wysvs-58qtp

**Review by fannycc — COMMENTED** [review]
LGTM. I have only comments about the legend as it looks like it's only related to the left map. It should be in the middle of the screen. Thanks!

**Comment by kfilip94** [comment]
Thanks @fannycc :+1:  I'll fix it in my next PR with biovar layers because I need to adjust the legend there anyway

File: src/layers.js [hunk 1]
```diff
@@ -126,97 +78,81 @@ export const vectorLayerCarto = (iso, species, scenario, year, opacity = 1) => {
...
+        species
+      }
+    },
+    layerConfig: {
+      account: 'simbiotica',
+      body: {
+        layers: [
+          {
+            options: {
+              sql: `WITH a AS (SELECT cartodb_id, the_geom_webmercator, uuid, iso3 FROM all_geometry {{where}}) SELECT a.the_geom_webmercator, a.cartodb_id, b.uuid, b.timeinterval, b.species, b.scenario, b.probabilityemca FROM ${iso.toLowerCase()}_zonal_spp_uuid as b INNER JOIN a ON b.uuid = a.uuid {{where2}}`
```
Comment (sorodrigo) [inline src/layers.js:101]: I'm probably missing something. Shouldn't we be filtering out by year === 1995?

**Comment by sorodrigo** [comment]
Great job!!!

**Comment by fannycc** [comment]
sorry just realized now. Two more things: (Maybe for other PRs?) 
the icons for zooming in and out are different than the ones in biovars, these are thicker dunno why, and in the biovar section we have the download button but in this view nope. Let me know if I should put pivotal tasks for these. thx!! :)

File: src/layers.js (hunk 1 above)
Comment (kfilip94) [reply src/layers.js:101]: We're not passing a scenario or year value, I hard-coded scenario value for current distributions:
image
and then I set this as a default for a scenario param, so filtering by year it's not needed here I think :wink: 
image

**Comment by kfilip94** [comment]
I'll fix these two issues in the biovars PR, thanks @fannycc for spotting this ;) If there are no other issues, I'll merge it into develop :)

**Pull request merged**